src/index/vespa_.py

```diff
@@ -240,18 +256,33 @@ def get_document_generator(
     physical_document_count = 0
     for path in paths:
         task = ParserOutput.model_validate_json(path.read_text())
- 
+
         task = filter_on_block_type(
             input=task, remove_block_types=config.BLOCKS_TO_FILTER
         )
 
-        task_array_file_path = cast(
-            Path, embedding_dir_as_path / f"{task.document_id}.npy"
-        )
-        embeddings = read_npy_file(task_array_file_path)
+        embeddings_by_model_slug = dict()
+
+        for model_slug in [
+            "baai-bge-small-en-v1-5",
+            "baai-bge-base-en-v1-5",
+            "msmarco-distilbert-base-tas-b",
+            "msmarco-distilbert-dot-v5",
```

Comment:
worth pulling out into a config so that we reduce points needing change when updating these

Reply:
I wanted to, but they're hardcoded a few lines down anyway. I couldn't think of a neat way of doing this, can you?

I don't think it's the end of the world if we need to make a PR here for a model change as we need to in infra anyway (it's hardcoded into the schema)

Reply:
yeah, good point, happy to ignore this than as it's hardcoded into input arguments anyways